rdt/transformers/categorical.py

```diff
@@ -329,7 +329,7 @@ def _fit(self, data):
         data = self._prepare_data(data)
 
         null = pd.isna(data)
-        self._uniques = list(pd.unique(data[~null]))
+        self._uniques = list(pd.unique(data[~null]))  # pylint: disable=E1130
```

Comment:
Same

Reply:
This is due to recent updates on `pylint` my older version was `pylint==2.12.2` but after updating it started failing. However instead of ignore those I would propose to fix them which is a minor change:

```python
null = pd.isna(data).to_numpy()
```